app/views/admin/creator_settings/_form.html.erb

```diff
@@ -83,9 +83,10 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <fieldset aria-describedby="section-description">
+  <fieldset aria-describedby="terms-description">
```

Comment:
I'm a bit confused by the change to `aria-describedby` (and the original value).  I'm looking at the DOM and I don't see an element with the ID `terms-description`.  That's not a blocker for this change, but perhaps an accessibility issue.

Reply:
Good catch. I've reverted this change, @jeremyf!